esmf_regrid/tests/unit/experimental/unstructured_scheme/test__cube_to_GridInfo.py

```diff
@@ -0,0 +1,108 @@
+"""Unit tests for miscellaneous helper functions in `esmf_regrid.experimental.unstructured_scheme`."""
+
+from iris.coords import DimCoord
+from iris.cube import Cube
+import numpy as np
+import scipy.sparse
+
+from esmf_regrid.esmf_regridder import Regridder
+from esmf_regrid.experimental.unstructured_scheme import _cube_to_GridInfo
+
+
+def _grid_cube_global(n_lats, n_lons):
```

Comment:
This code is very similar to below.  Let's simplify it quickly, and use our usual Iris ordering for the inputs
`def _grid_cube(n_lons, lon_st, lon_end, n_lats, lat_st, lat_end)`

Reply:
Much better 👍